### PR TITLE
Accessibility Issues - Put back titles for missing pages issue #1441

### PR DESCRIPTION
**Pull request by bhavi**
Fixes # .

Changes proposed in this PR:
-Some of the page titles went missing as part of merge process for the issue #1441. Putting them back as part of this PR.

### CODE REVIEW

**Comment by briri** [comment]
looks fine

File: app/views/super_admin/notifications/edit.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<% title 'Editing Notification' %>
```
Comment (xsrust) [inline app/views/super_admin/notifications/edit.html.erb]: needs gettext markup _(' ')

File: app/views/super_admin/notifications/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<% title 'Notifications' %>
```
Comment (xsrust) [inline app/views/super_admin/notifications/index.html.erb]: needs gettext markup _(' ')

File: app/views/super_admin/notifications/new.html.erb [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<% title 'New Notification' %>
```
Comment (xsrust) [inline app/views/super_admin/notifications/new.html.erb]: needs gettext markup _(' ')

File: app/views/usage/index.html.erb [hunk 4]
```diff
@@ -1,3 +1,5 @@
+<% title 'Usage statistics' %>
```
Comment (xsrust) [inline app/views/usage/index.html.erb]: needs gettext markup _(' ')

File: app/views/users/admin_grant_permissions.html.erb [hunk 5]
```diff
@@ -1,3 +1,5 @@
+<% title 'Edit User Privileges' %>
```
Comment (xsrust) [inline app/views/users/admin_grant_permissions.html.erb]: needs gettext markup _(' ')

File: app/views/users/admin_index.html.erb [hunk 6]
```diff
@@ -1,4 +1,5 @@
 <% title 'User accounts' %>
```
Comment (xsrust) [inline app/views/users/admin_index.html.erb]: needs gettext markup _(' ')

**Pull request merged**